public/client.js

```diff
@@ -514,22 +514,26 @@ function formatSizeUnits(size){
     return giga.toFixed(1) + " GB";
 }
 //makes all unrecognized files downloadable
-function unknown_file_dowloadable(attached){
+function unknown_file_downloadable(attached){
     let thegotten = getFileSize(attached)
-    return `<a href='${attached}'>${thegotten[1].split('"')[1]+" ("+thegotten[0]+")"}</a>`
+    return `<div class="downloadable_file_div"><a class="downloadable_file" target='_blank' href='${attached}'>${thegotten[1].split('"')[1]+" ("+thegotten[0]+")"}<img class="embed-dl-img" src='./assets/img/download_icon.svg' width=30></a>`
+}
+
+function adddedlbutton(attache){
+    return `<a style="padding-left: 1rem;" href='${attache}' target="_blank" rel="noreferrer noopener"><img src='./assets/img/download_icon.svg' width=30></a>`;
```

Comment:
```suggestion
    return `<a style="padding-left: 1rem;" href='${attache}' target="_blank" rel="noreferrer noopener"><i class="fa-solid fa-download"></i></a>`;
```
You don't appear to have included the "download icon" SVG in this commit.
That being said, Quarky (*very legally*) uses Font Awesome, so you can just use their icons instead.

Reply:
oopsie